packages/server/graphql/mutations/endNewMeeting.ts

```diff
@@ -307,7 +357,13 @@ export default {
     const presentMemberUserIds = presentMembers.map(({userId}) => userId)
     endSlackMeeting(meetingId, teamId, dataLoader).catch(console.log)
 
+    const pinnedAgendaItems = await getPinnedAgendaItems(teamId)
     const result = await finishMeetingType(completedMeeting, dataLoader)
+
+    pinnedAgendaItems.map((agendaItem) => {
+      clonePinnedAgendaItem(agendaItem, viewerId)
```

Comment:
this will result in multiple DB hits. can you reduce it to a single bulk insert?

Reply:
Thanks - I've updated this 👍